.github/workflows/conda-build.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will setup a conda build for dependency checks by pulling in the codebuild docker images 
+# and will execute only on modification of yml files for PRs and merges.
+
+name: Build Conda Environment
+
+on:
+  push:
+    branches:
+      - main
+    paths: 
+      - '*.yml'
+  pull_request:
+    branches:
+      - main
+      - feature/**
+    paths: 
+      - '*.yml'
```

Comment:
Nit: This will also run on github workflow changes. Is that OK?

Reply:
I checked that it doesn't run on changes in github workflow changes. Probably since it is in .github folder.